src/domains/userProjectRole/repository/userProjectRole.repository.impl.ts

```diff
@@ -35,4 +35,24 @@ export class UserProjectRoleRepositoryImpl implements UserProjectRoleRepository
 
     return userProjectRole === null ? null : new UserProjectRoleDTO(userProjectRole);
   }
+
+  /**
+   * Updates a user-project role relationship by ID.
+   * @param input Object containing data to update a userProjectRole.
+   * @returns A Promise resolving to a UserProjectRoleDTO object.
+   */
+  async update(input: UserProjectRoleUpdateExtended): Promise<UserProjectRoleDTO> {
```

Comment:
I think this method isn't used anywhere. Please, remove it and all related DTO and classes.

Reply:
Yes, it is used in line 94 in src/domains/user/service/user.service.impl.ts